### ISSUE TITLE
Tensorflow 2.0

### ISSUE THREAD

**Issue by rexdouglass** [hanxiao/bert-as-service#266]
Installed Tensorflow 2.0 today and tried bert-as-service (BAS) which throws Tensorflow>=1.10 error. If tensorflow 2.0 doesn't break BAS, then update the assert to allow it. If it does break BAS, then documentation and error update to "Tensorflow >=1.10 and <2.0 required!"

bert-serving-start -model_dir /home/Downloads/cased_L-24_H-1024_A-16  -num_worker=1  -pooling_strategy NONE -max_batch_size 256 -gpu_memory_fraction 0.9
Traceback (most recent call last):
  File "/usr/local/bin/bert-serving-start", line 6, in <module>
    from bert_serving.server.cli import main
  File "/usr/local/lib/python3.6/dist-packages/bert_serving/server/__init__.py", line 29, in <module>
    _tf_ver_ = check_tf_version()
  File "/usr/local/lib/python3.6/dist-packages/bert_serving/server/helper.py", line 160, in check_tf_version
    assert int(tf_ver[0]) >= 1 and int(tf_ver[1]) >= 10, 'Tensorflow >=1.10 is required!'
AssertionError: Tensorflow >=1.10 is required!

**Prerequisites**
> Please fill in by replacing `[ ]` with `[x]`.

* [ x] Are you running the latest `bert-as-service`?
* [ x] Did you follow [the installation](https://github.com/hanxiao/bert-as-service#install) and [the usage](https://github.com/hanxiao/bert-as-service#usage) instructions in `README.md`?
* [ x] Did you check the [FAQ list in `README.md`](https://github.com/hanxiao/bert-as-service#speech_balloon-faq)?
* [ x] Did you perform [a cursory search on existing issues](https://github.com/hanxiao/bert-as-service/issues)?

**Comment by hanxiao** [comment]
fyi, this issue is fixed in #271 and the new feature is available since 1.8.4, please do
```bash
pip install -U bert-serving-server bert-serving-client
```
for the update.

**Comment by SamyAteia** [comment]
Current version of bert-serving-server on pip is 1.10.0
```
pip3 show bert-serving-server
Name: bert-serving-server
Version: 1.10.0
Summary: Mapping a variable-length sentence to a fixed-length vector using BERT model (Server)
Home-page: https://github.com/hanxiao/bert-as-service
Author: Han Xiao
Author-email: artex.xh@gmail.com
License: MIT
Location: /usr/local/lib/python3.8/dist-packages
Requires: numpy, pyzmq, six, termcolor, GPUtil
Required-by:
```

How to get 1.8.4?
